fe/fe-core/src/main/java/com/starrocks/analysis/ColumnDef.java

```diff
@@ -362,6 +366,19 @@ public static void validateDefaultValue(Type type, Expr defaultExpr) throws Anal
             if (FunctionSet.NOW.equalsIgnoreCase(functionName) && type.getPrimitiveType() != PrimitiveType.DATETIME) {
                 throw new AnalysisException(String.format("Default function now() for type %s is not supported", type));
             }
+            // default function uuid currently only support VARCHAR type.
```

Comment:
VARCHAR + CHAR + STRING?

Reply:
CHAR is not recommended for users, so it is not supported first, but STRING can be supported.